docs/topics/mta-cli-analyze-unsupported-provider.adoc

```diff
@@ -0,0 +1,24 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-07-30
+:_mod-docs-content-type: PROCEDURE
+
+[id="mta-cli-analyze-unsupported-provider_{context}"]
+= Analyzing a multi-language application for an unsupported language provider
+
+When analyzing a multi-language application with {ProductFullName} {CLINameTitle}, you can run the analysis for an unsupported language provider. To do so, you must override an existing supported language provider with your own unsupported language provider by using the `--provider-override` option. 
+
+IMPORTANT: You must create a configuration file for your unsupported language provider before overriding the supported provider.
+
+
+.Prerequisites
+
+* You created a configuration file for your unsupported language provider.
+
+.Procedure
+
+* Override an existing supported language provider with your unsupported provider:
++
+[source,terminal,subs="attributes+"]
+----
+$ mta-cli analyze --provider-override <path_to_configuration_file> --output=<path_to_the_output_directory> --rules <path_to_custom_rules>
+----
```

Comment:
The case hasn't been tested yet, thus, this procedure will be changed after testing is complete.

Reply:
Hi @eemcmullan,
I guess this should be sufficient to provide, right?

Reply:
+1 once the flag name is updated from the above comment.

Reply:
@eemcmullan, fixed the option name in the suggestion above, is that ok now?

Reply:
One small NIT. Otherwise LGTM